packages/wrangler/src/d1/delete.ts

```diff
@@ -38,7 +41,10 @@ export const d1DeleteCommand = createCommand({
 			name
 		);
 
-		logger.log(`About to delete DB '${name}' (${db.uuid}).`);
+		logger.log(
+			`\nAbout to delete ${chalk.bold("remote")} database DB '${name}' (${db.uuid}).\n` +
```

Comment:
I wonder if we should add a double newline to the end of `printResourceLocation("remote")` instead. 

Taking a look at how `wrangler.js d1 migrations list` renders, the local one prints a double-newline after the message, but the remote one doesn't. I think I would prefer it with the double-newline in both cases.

<img width="437" height="285" alt="Image" src="https://github.com/user-attachments/assets/aeb69062-b5e9-40c1-9388-0b96bc4a8455" />